### DISCUSSION TITLE
Is downgrade available for self-hosted version?

### DISCUSSION THREAD

**Discussion by skabbit** [langfuse/langfuse#4361] [Support] [answered]
I'm experiencing issue with 2.91 OSS: https://github.com/langfuse/langfuse/issues/4357
Is there any backward migration to be applied, if I revert docker image to 2.90?
Is this reliable in any sense?

**Comment by maxdeichmann** [comment, marked as answer]
Hi, the fix for the issue was released in https://github.com/langfuse/langfuse/releases/tag/v2.92.0
